### PR TITLE
Add Scrolls latest

### PR DESCRIPTION
**Pull request by ESWAT**
Scrolls is a trading card game from @mojang.

### CODE REVIEW

File: Casks/scrolls.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask :v1 => 'scrolls' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://download.scrolls.com/new-installer/mac/Scrolls.dmg'
+  name 'scrolls'
+  homepage 'https://scrolls.com/'
+  license :commercial
```
Comment (vitorgalvao) [inline Casks/scrolls.rb]: Is the app paid?

File: Casks/scrolls.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+cask :v1 => 'scrolls' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://download.scrolls.com/new-installer/mac/Scrolls.dmg'
+  name 'scrolls'
```
Comment (vitorgalvao) [inline Casks/scrolls.rb]: Capitalisation matters in `name`. Shouldn’t it be “Scrolls”?

File: Casks/scrolls.rb (hunk 1 above)
Comment (ESWAT) [inline Casks/scrolls.rb]: Yes. The same installer includes a demo mode if you don’t use a paid account, but that does not seem right for `:freemium`

Comment (vitorgalvao) [inline Casks/scrolls.rb]: It shouldn’t be `:freemium`, but `:gratis`. What matters is [the license of the binary, not the business model of the company](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#license-stanza-details).

Comment (ESWAT) [inline Casks/scrolls.rb]: Alright, thanks for the clarification!

**Comment by vitorgalvao** [comment]
Looks good now. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by fanquake** [comment]
Thanks

**Pull request merged**